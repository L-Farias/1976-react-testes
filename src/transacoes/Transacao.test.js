import React from 'react';
import {render} from '@testing-library/react';
import Transacao from "./Transacao";

describe('Componente de transação de extrato' , () =>{
    it('O snapshot do compenent deve permanecer sempre o mesmo', 
    () => {
        const {container} =  render(<Transacao
        data="14/10/2021"
        tipo="saque"
        valor = "20.00"
        />)
      expect(container.firstChild).toMatchSnapshot();
    })
});