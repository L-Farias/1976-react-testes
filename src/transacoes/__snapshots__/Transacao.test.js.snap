// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação de extrato O snapshot do compenent deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    14/10/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
